testing/environments/infra/terraform/modules/benchmark_executor/README.md

```diff
@@ -0,0 +1,52 @@
+<!-- BEGIN_TF_DOCS -->
+# Benchmark executor module
+
+This module creates a benchmark executor virtual machine in AWS on the specified region
```

Comment:
It seems like there are a bunch of Terraform files like this one copied from APM, do we actually need all of them? Can we keep only the ones that are actually necessary?